docs/README.md

```diff
@@ -45,6 +45,16 @@ val result2 = for {
 } yield (encoded, decoded)
 ```
 
+5. (Optional) You can use a tagged type to use `Instant` for DynamoDB's _TimeToLive_ which is based on epoch seconds.
```

Comment:
Since this is so DynamoDB-specific, you could also link to the docs that state this 😊 
```suggestion
5. (Optional) You can use a tagged type to use `Instant` for DynamoDB's _TimeToLive_ which [is based on epoch seconds](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/time-to-live-ttl-before-you-start.html#time-to-live-ttl-before-you-start-formatting).
```